docs/_layouts/default.html

```diff
@@ -5,29 +5,29 @@
+		<link rel="icon" sizes="128x128" href="{{ site.baseurl }}/assets/icons/favicon-128x128.png?version={{ site.github.build_revision }}">
+		<link rel="icon" sizes="96x96" href="{{ site.baseurl }}/assets/icons/favicon-96x96.png?version={{ site.github.build_revision }}">
+		<link rel="icon" sizes="64x64" href="{{ site.baseurl }}/assets/icons/favicon-64x64.png?version={{ site.github.build_revision }}">
+		<link rel="icon" sizes="48x48" href="{{ site.baseurl }}/assets/icons/favicon-48x48.png?version={{ site.github.build_revision }}">
+		<link rel="icon" sizes="32x32" href="{{ site.baseurl }}/assets/icons/favicon-32x32.png?version={{ site.github.build_revision }}">
+		<link rel="icon" sizes="16x16" href="{{ site.baseurl }}/assets/icons/favicon-16x16.png?version={{ site.github.build_revision }}">
+		<link rel="apple-touch-icon" sizes="57x57" href="{{ site.baseurl }}/assets/icons/apple-touch-icon-57x57.png?version={{ site.github.build_revision }}">
+		<link rel="apple-touch-icon" sizes="76x76" href="{{ site.baseurl }}/assets/icons/apple-touch-icon-76x76.png?version={{ site.github.build_revision }}">
+		<link rel="apple-touch-icon" sizes="120x120" href="{{ site.baseurl }}/assets/icons/apple-touch-icon-120x120.png?version={{ site.github.build_revision }}">
+		<link rel="apple-touch-icon" sizes="152x152" href="{{ site.baseurl }}/assets/icons/apple-touch-icon-152x152.png?version={{ site.github.build_revision }}">
+		<link rel="apple-touch-icon" sizes="167x167" href="{{ site.baseurl }}/assets/icons/apple-touch-icon-167x167.png?version={{ site.github.build_revision }}">
+		<link rel="apple-touch-icon" sizes="180x180" href="{{ site.baseurl }}/assets/icons/apple-touch-icon-180x180.png?version={{ site.github.build_revision }}">
+		<link rel="stylesheet" type="text/css" href="{{ site.baseurl }}/assets/css/normalize.css?version={{ site.github.build_revision }}" media="screen">
 		<link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Open+Sans:400,400i,700,700i" media="screen">
-		<link rel="stylesheet" type="text/css" href="{{ site.baseurl }}/assets/css/stylesheet.css?version=df12dbc" media="screen">
+		<link rel="stylesheet" type="text/css" href="{{ site.baseurl }}/assets/css/stylesheet.css?version={{ site.github.build_revision }}" media="screen">
 		<script src="{{ site.baseurl }}/assets/js/vendor/jquery-3.2.1.min.js"></script>
 		<script src="{{ site.baseurl }}/assets/js/vendor/d3.v4.min.js"></script>
 		<script src="{{ site.baseurl }}/assets/js/vendor/moment-with-locales.min.js"></script>
 		<script src="{{ site.baseurl }}/assets/js/vendor/Chart-2.7.1.min.js"></script>
 		<script src="{{ site.baseurl }}/assets/js/vendor/spin-2.3.2.min.js"></script>
-		<script src="{{ site.baseurl }}/assets/js/charts.js?version=1ff0187"></script>
+		<script src="{{ site.baseurl }}/assets/js/charts.js?version={{ site.github.build_revision }}"></script>
```

Comment:
Is there any value to add this flag to the other (already versioned!) libraries? Maybe `moment-with-locales.min.js`?

Reply:
OK. That's an independent change. I'll merge is as-is.